ci-operator/config/openstack-k8s-operators/mariadb-operator/openstack-k8s-operators-mariadb-operator-master.yaml

```diff
@@ -8,7 +8,27 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.18
+    tag: rhel-8-release-golang-1.19-openshift-4.12
+images:
+- dockerfile_literal: |
```

Comment:
This literal Dockerfile will be replaced by images generated in openstack-k8s-operators-ci repo, in a follow up PR.

Reply:
sounds good. It might be nice to have these settings on a per repo bases too. As we sometimes bump the dependencies below to new versions (bumping golang or operator-sdk for example)